package.json

```diff
@@ -41,6 +42,7 @@
     "compare-fidelity": "./scripts/compare-fidelity-to-ref.sh",
     "install-renderers": "./scripts/install-third-party-renderers.sh",
     "update-screenshots": "node ./scripts/update-screenshots.js",
+    "checkout-samples": "./scripts/checkout-khronos-gltf-samples.sh",
```

Comment:
nit, maybe `get-samples` or `fetch-samples` is a bit more obvious at what this does

Reply:
Will fix 👍